package.json

```diff
@@ -29,7 +29,7 @@
     "db:local:generate_types": "supabase gen types typescript --local --schema public > src/databaseTypesFile.ts",
     "db:remote:generate_types": "supabase gen types typescript --schema public > src/databaseTypesFile.ts",
     "db:generate_seed": "npx tsx supabase/seed/seed.mts > supabase/seed.sql",
-    "post_checkout": "npm install && npx supabase stop && npx supabase start && npm run dev:reset_supabase && npx snaplet generate"
+    "post_checkout": "npm install && npx supabase start && npm run dev:reset_supabase && npx snaplet generate"
```

Comment:
Can you revert the changes in this file please?:)

Reply:
Done!